web/modules/custom/dept_migrate/src/MigrateUuidLookupManager.php

```diff
@@ -325,6 +325,11 @@ public function lookupBySourceFileId(array $fids) {
 
       // Switch table if there's a specified type for this file.
       switch ($file['d7type']) {
+        case 'document':
```

Comment:
This explains a lot about why the file lookups for documents returned strange results. Oops